launcher/ui/dialogs/CreateShortcutDialog.ui

```diff
@@ -454,6 +454,22 @@
     </hint>
    </hints>
   </connection>
+  <connection>
+   <sender>joinWorldCheckBox</sender>
+   <signal>toggled(bool)</signal>
+   <receiver>CreateShortcutDialog</receiver>
+   <slot>updateDialogState()</slot>
+   <hints>
+    <hint type="sourcelabel">
+     <x>140</x>
+     <y>59</y>
+    </hint>
+    <hint type="destinationlabel">
+     <x>397</x>
+     <y>164</y>
+    </hint>
+   </hints>
+  </connection>
```

Comment:
Please do not add more of these, connect the signals directly in code

Reply:
And ideally using the new style syntax, like here:
![image](https://github.com/MultiMC/Launcher/assets/203326/30f6df54-ed13-47f7-bf35-0cd833812c57)